### DISCUSSION TITLE
Running The Examples for in Quickstart or Examples

### DISCUSSION THREAD

**Discussion by BrianBut** [dtiesling/flask-muck#63] [Q&A]
I have tried to run the examples referred to https://github.com/dtiesling/flask-muck, and flask-muck/examples/00_quickstart/app.py.
In neither case can I get it to find the routes. i.e "GET /api/teachers HTTP/1.1" 404 -" or "POST /api/v1/todos HTTP/1.1" 404.
flask routes generates only:

Endpoint                 Methods  Rule                            
-----------------------  -------  --------------------------------
flasgger.<lambda>        GET      /apidocs/index.html             
flasgger.apidocs         GET      /apidocs/                       
flasgger.apispec_1       GET      /apispec_1.json                 
flasgger.oauth_redirect  GET      /oauth2-redirect.html           
flasgger.static          GET      /flasgger_static/<path:filename>
static                   GET      /static/<path:filename>       

What am I missing?

**Comment by dtiesling** [comment]
Sorry about that @BrianBut, I had let the docs get stale for that example. Try out the updated instructions that have the corrected routes - https://github.com/dtiesling/flask-muck/tree/main/examples/00_quickstart

**Comment by dtiesling** [comment]
I'm going to take a pass on the other 2 examples as well and make sure they are up to date.
